### ISSUE TITLE
Two particle containers are rendered incorrectly, if rotation is enabled

### ISSUE THREAD

**Issue by sergebat** [pixijs/pixi.js#1953]
The following fiddle displays two bunnies side-by-side in two separate ParticleContainer instances:
http://jsfiddle.net/sergebat/wd03Lxt8/1/

However, if second particle container allows dynamic rotation for its kids, things in the first ParticleContainer got distorted in Chrome and Firefox at least on OSX:
http://jsfiddle.net/sergebat/6w9m9hft/1/

In canvas mode everything works like a charm:
http://jsfiddle.net/sergebat/yxzpcsp9/1/

Chrome version 43.0.2357.130 (64-bit) on OSX 10.9.5). Version: Pixi.js 3.0.7.

Screenshot:
<img width="438" alt="screenshot 2015-07-06 00 12 41" src="https://cloud.githubusercontent.com/assets/5421460/8513253/5c3c9c3c-236b-11e5-8789-062ee6798cea.png">

**Comment by sergebat** [comment]
I'd like to add that rotation is 0 for both sprites in all scenarios above. 

In actual game code rotation seem to work fine in the second ParticleContainer. For now I replaced ParticleContainer with Container, and it seems to work ok. 

Use-case I was hoping to implement with two particle containers is:
- Container 1: Game background (sky + mountains), only positions change.
- Container 2: Game content, positions/rotation will have to change (and new sprites are sometimes added).

**Comment by sergebat** [comment]
I've experimented some more. The problem also goes away, if both particle containers allow rotation, hope this helps: 
http://jsfiddle.net/sergebat/wd03Lxt8/2/

**Comment by rejemy** [comment]
I've also reproduced this error in both 3.0 and 4.0dev. It happens whenever you have two different particle containers that have different dynamic properties from each other. I've started to investigate into it, I suspect the wrong buffer is being left bound during an upload, causing vertex attributes to be written out of alignment into the wrong buffer. There's a lot of code to come up to speed on so I don't know how long it will take me to figure it out.
